doc/neps/nep-0029-deprecation_policy.rst

```diff
@@ -74,11 +74,19 @@ a project with a major or minor version released in December 2019 should
 support Python 3.6 and newer, and a project with a major or minor version
 release in November 2020 should support Python 3.7 and newer.
 
-The current Python release cadence is 18 months so a 42 month window
-ensures that there will always be at least two minor versions of Python
-in the window.  The window is extended 6 months beyond the anticipated two-release
-interval for Python to provide resilience against small fluctuations /
-delays in its release schedule.
+When this NEP was drafted the Python release cadence was 18 months so a 42
+month window ensured that there would always be at least two minor versions of
+Python in the window.  The window was extended 6 months beyond the anticipated
+two-release interval for Python to provide resilience against small
+fluctuations / delays in its release schedule.
+
+The Python release cadence was increased to 12 months in `PEP 0602
```

Comment:
This is a bit nitpicky, but I'd say cadence is frequency, whereas you are referring to interval (/period) here.  So, I'd have used `The Python release interval was decreased to 12 months"

Reply:
Yeah, writing "increased to a --smaller number-- " felt a bit weird..

How about: "The Ptyhon release cadenced increased and releases are now every 12 months pre `PEP...`"

Reply:
"increased, with releases now every 12 months" maybe? Anyway 👍